api/clock/api.go

```diff
@@ -36,27 +37,37 @@ func (a *API) loc() *time.Location {
 }
 
 // Configure for clock
-func (a *API) Configure(body types.ConfigMessage) {
+func (a *API) Configure(body types.ConfigMessage) error {
 	a.ConfigurePosition(body.Position)
 	log.Println("Configuring CLOCK:", body)
 	oldConfig := a.Config
 
 	if err := json.Unmarshal(body.Config, &a.Config); err != nil {
 		log.Println("Could not properly configure clock")
 		a.Config = oldConfig
-		return
+		return errors.New("could not properly configure clock")
```

Comment:
Might eventually want to create our own error types, but that's for another day.